d2common/d2enum/game_event.go

```diff
@@ -0,0 +1,118 @@
+package d2enum
+
+// GameEvent represents an envent in the game engine
+type GameEvent int
+
+const (
+	// ToggleGameMenu will display the game menu
+	ToggleGameMenu GameEvent = iota + 1
+
+	// ToggleCharacterPanel toggles the player's character sheet
+	ToggleCharacterPanel
+	// ToggleInventoryPanel toggles the player's inventory
+	ToggleInventoryPanel
```

Comment:
to appease the linter, you could have put `// Game events` above the const declaration line.